docs/types.rst

```diff
@@ -0,0 +1,506 @@
+.. highlight:: yaml
+
+.. _types:
+
+Types
+^^^^^
+
+Node Types
+==========
+
+.. cfy:node:: cloudify.azure.nodes.ResourceGroup
+
+This example shows adding additional parameters, and explicitly defining the azure_config::
```

Comment:
Sounds like a good option to me. If words are not explicitly required for the audience, they just clutter the important content.